.github/workflows/test-all.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
           [
             "ubuntu:focal",
             "ubuntu:jammy",
+            "ubuntu:noble",
```

Comment:
As mentioned in https://github.com/devcontainers/features/pull/969/files#r1598718691, can we use `mcr.microsoft.com/devcontainers/base:noble` instead ?

Reply:
https://github.com/devcontainers/features/pull/971/commits/6b6ad16487edbe685c598ed57c0ada36995648df